domain/src/main/kotlin/no/nav/su/se/bakover/domain/kontrollsamtale/Kontrollsamtale.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.su.se.bakover.domain.kontrollsamtale
+
+import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.common.endOfMonth
+import no.nav.su.se.bakover.common.periode.Periode
+import no.nav.su.se.bakover.common.startOfMonth
+import no.nav.su.se.bakover.common.zoneIdOslo
+import no.nav.su.se.bakover.domain.vedtak.Vedtak
+import java.time.Clock
+import java.time.LocalDate
+import java.util.UUID
+
+data class Kontrollsamtale(
+    val id: UUID = UUID.randomUUID(),
+    val opprettet: Tidspunkt = Tidspunkt.now(),
+    val sakId: UUID,
+    val innkallingsdato: LocalDate?,
+    val status: Kontrollsamtalestatus,
+    val frist: LocalDate?
+) {
+    companion object {
+        fun fromVedtak(vedtak: Vedtak): Kontrollsamtale {
+            val innkallingsdato = regnUtInnkallingsdato(vedtak.periode, vedtak.opprettet.toLocalDate(zoneIdOslo))
+
+            return Kontrollsamtale(
+                sakId = vedtak.behandling.sakId,
+                innkallingsdato = innkallingsdato,
```

Comment:
innkallingsdato kan jo være null, dersom den er for nær stønadsslutt. Da er det vel `SKAL_IKKE_INNKALLES` eller noe slikt? :shrug:

Reply:
Godt poeng. Rent funksjonelt vil det ikke endre noe, men gir litt bedre oversikt i dataene muligens

Reply:
Foreløpig så blir det kun opprettet en Kontrollsamtale under iverksett. Dersom innkallingsdato er null, så er tanken at det ikke blir opprettet en kontrollsamtale i databasen. Derfor er det _eeeegentlig_ ikke bruk for en sånn status. Dog kan det hende vi trenger en "negativ" status som Jacob også hinter til, dersom et allerede opprettet innslag må "annuleres"